tests/ruff.toml

```
@@ -7,6 +7,7 @@ extend-ignore = [
     "B904", # Use raise from to specify exception cause
     "N815", # Variable {name} in class scope should not be mixedCase
     "RUF018", # Avoid assignment expressions in assert statements
+    "SLF", # Private member accessed: Tests do often test internals a lot
```

While this is correct (as this ruleset contains a single rule at this point), I do think we need to be specific here. Simply to ensure/check for behavior when for example another rule is added to this set in the future.

```suggestion
    "SLF001", # Private member accessed: Tests do often test internals a lot
```

---

But then we would need to expand all rule groups into single rules when following this for all 🤔

---

All ignore rules in here are already expanded...

---

Oh, sorry, I was on mobile and didn't check in which file your review exactly was :see_no_evil: